components/dashboard/script.js

```diff
@@ -0,0 +1,26 @@
+let fs = require('fs');
+
+let vars = {
+    REACT_APP_HOSTNAME : "Gitpod",
+    REACT_APP_HOST_URL : "http://developarm.devspaces.com/",
+    REACT_APP_DOCS_URL : "https://www.gitpod.io/docs/",
+    REACT_APP_SOCIAL_URL : "https://community.gitpod.io/",
+    REACT_APP_FAVICON_192 : "$REACT_APP_HOST_URL/icons/favicon192.png",
+    REACT_APP_FAVICON_256 : "$REACT_APP_HOST_URL/icons/favicon256.png",
+    REACT_APP_FAVICON_512 : "$REACT_APP_HOST_URL/icons/favicon512.png",
+    REACT_APP_BRAND_SVG : "$REACT_APP_HOST_URL/icons/gitpod.svg",
+    REACT_APP_LOGO : "$REACT_APP_HOST_URL/icons/gitpod32.svg"
+};
+
+let data = "";
+for(let prop in vars) data += prop + "=" + vars[prop] + "\n";
```

Comment:
<details>
<summary>:warning: <strong>This code contains single line control structures without curly braces, `{}`.</strong> (click for details)</summary>

Consider surrounding these statements with `{ }`.  
  
Braces are used around all statements, even singletons, when they are part of a control structure, such as an `if`-`else` or `for` statement.  
  
This makes it easier to add statements without accidentally introducing bugs due to forgetting to add braces. Readability is also significantly improved.

**[More details](https://confluence.devfactory.com/x/fHvlFQ)**

## The improvement suggestion in this case
```suggestion
for (let prop in vars) {
  data += prop + "=" + vars[prop] + "\n";
}
```  
> NOTE: Consider applying the structure of this solution in other parts of the code, if there is another occurrence(s).
</details>

[//]: # (<a id="577f3a94-c7d6-498b-bc93-fda1fe27cfce" data-finding-id="367360892" href="none" codereview-uuid-reserved-tag data-business-id="507be860-41ae-46a0-83e9-7f8e359e3039" data-analyzer="js" />)